cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,184 @@
+/// <reference types='cypress' />
+
+const { generateUser } = require('../support/generate');
+const { firstName, lastName, email, age, salary, department } = generateUser();
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('should contain pagination', () => {
+    cy.get('.-pagination')
+      .should('contain', 'Previous')
+      .and('contain', 'Next')
+      .and('contain', 'Page')
+  });
+
+  it('should provide to change rows count selection', () => {
+    cy.get('select')
+      .select('5 rows')
+      .should('have.value', '5');
+
+    cy.get('select')
+      .select('10 rows')
+      .should('have.value', '10');
+
+    cy.get('select')
+      .select('20 rows')
+      .should('have.value', '20');
+
+    cy.get('select')
+      .select('25 rows')
+      .should('have.value', '25');
+
+    cy.visit('/')
+      .get('select')
+      .select('50 rows')
+      .should('have.value', '50');
+
+    cy.visit('/')
+      .get('select')
+      .select('100 rows')
+      .should('have.value', '100');
```

Comment:
DRY!
You've written 6 times the same. It will be good to write a function for changing the rows